Makefile

```diff
@@ -96,6 +96,14 @@ endif
 # Optional file including user-specific settings (HUB, TAG, etc)
 -include .istiorc.mk
 
+# Base directory for installing artifacts
+INSTALL_PREFIX ?= /usr/local
```

Comment:
Maybe time to open the discussion on where we want istio to be installed - /opt/istio or /usr/local/istio or /usr/local/bin. And next if we include version in name (/opt/istio-0.8).

Reply:
Yeah, should we raise the question on ML? Or just ping relevant people here?

Reply:
mailing list seems appropriate, but as long as it's configurable it doesn't matter too much
we can debate the default location for a while I suppose; my take is anything that isn't to be invoked directly by a user should be in /usr/local/istio (or $INSTALL_PREFIX/istio) and any command that may be invoked by a user should have a name unlikely to conflict with anything (e.g istioctl is good, fortio is good; agent, client,... are bad)